adafruit_lis2mdl.py

```diff
@@ -91,7 +89,31 @@ class LIS2MDL:  # pylint: disable=too-many-instance-attributes
     """
     Driver for the LIS2MDL 3-axis magnetometer.
 
-    :param busio.I2C i2c_bus: The I2C bus the LIS2MDL is connected to.
+    :param ~busio.I2C i2c:  The I2C bus the device is connected to
+
+    **Quickstart: Importing and using the device**
+
+        Here is an example of using the :class:`LIS2MDL` class.
+        First you will need to import the libraries to use the sensor
+
+        .. code-block:: python
+
+            import board
+            import adafruit_lis2mdl
+
+        Once this is done you can define your `board.I2C` object and define your sensor object
+
+        .. code-block:: python
+
+            i2c = board.I2C()
```

Comment:
Same question.

Reply:
Changes in https://github.com/adafruit/Adafruit_CircuitPython_LIS2MDL/pull/13/commits/a5a8dfb3113f35f419452c2715b7ff99037e39c4